src/main/java/practice/CandidateValidator.java

```diff
@@ -1,5 +1,23 @@
 package practice;
 
-public class CandidateValidator {
+import java.util.function.Predicate;
+import model.Candidate;
+
+public class CandidateValidator implements Predicate<Candidate> {
+    private static final int MIN_AGE = 35;
+    private static final int MIN_YEAR_IN_UA = 10;
+    private static final String BEST_NATIONALITY = "Ukrainian";
+
     //write your code here
+    @Override
+    public boolean test(Candidate candidate) {
```

Comment:
```suggestion
    public boolean test(Candidate candidate) {
    String[] periodInUkr = person.getPeriodsInUkr().split(DATE_SPLIT_REGEX);
    int periodDuration = Integer.parseInt(periodInUkr [YEAR_TO_INDEX])
      - Integer.parseInt(periodInUkr [YEAR_FROM_INDEX]);
    boolean livedInUrkEnough = periodDuration >= MIN_YEAR_IN_UA;
```
Use this boolean and remove IF